src/EnergyPlus/UnitarySystem.cc

```diff
@@ -3785,7 +3787,10 @@ namespace UnitarySystems {
 
                         // Get the Heating Coil hot water max volume flow rate
                         thisSys.MaxHeatCoilFluidFlow = WaterCoils::GetCoilMaxWaterFlowRate("Coil:Heating:Water", loc_m_HeatingCoilName, errFlag);
-                        if (thisSys.MaxHeatCoilFluidFlow == DataSizing::AutoSize) thisSys.m_RequestAutoSize = true;
+                        if (thisSys.MaxHeatCoilFluidFlow == DataSizing::AutoSize) {
+                            thisSys.m_RequestAutoSize = true;
+                            thisSys.m_DesignHeatingCapacity = DataSizing::AutoSize;
+                        }
```

Comment:
OK, so just setting the heating capacity to autosize everywhere in this setup function, not just setting the autosize flag.  Fine.

Reply:
Well, the intent was to set thisSys.m_DesignHeatingCapacity = DataSizing::AutoSize; only for coils that did not have a Capacity input. The Coil:Heating:Water does have a capacity input field, although it's hard for water coils to know the design capacity for the alternate sizing method of UA and water flow rate (during GetInput). I wonder if this is why the diff's file has differences?